### PR TITLE
FS-4317-no-js-on-funding-service-dasboard-page

### PR DESCRIPTION
**Pull request by ramsharma-prog**
### Ticket
https://dluhcdigital.atlassian.net/jira/software/c/projects/FS/boards/50?selectedIssue=FS-4317

### Description
The investigation aimed to identify a bug disrupting JS functionality within the accordion feature on the dashboard summary page. While the version was incremented from 2.5.0 to 2.8.0, it was not escalated to 3.0.0 due to the latter's known issue of breaking accordion functionality. A pull request (PR) addressing version 3.0.0 has been linked, requiring additional investigation by the govuk-frontend jinja team to resolve the issue stemming from the aria-labelledby class.

### Issue to be resolved by govuk frontend team
https://github.com/alphagov/govuk-frontend/pull/4628

### Ticket raised to fix the issue once the issue is fixed by govuk frontend team
https://dluhcdigital.atlassian.net/browse/FS-4330

### File needs fixing post version bumped from 2.8.0
https://github.com/communitiesuk/funding-service-design-assessment/blob/FS-4317-no-js-on-funding-service-dasboard-page/app/blueprints/assessments/templates/assessor_tool_dashboard.html#L63

### CODE REVIEW

File: app/static/src/js/helpers.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ if (showTagsElement) {
 
 // This is a temporary fix to ensure tab changing works for flag history.
 // Updating to `4.7.0` for govuk css broke this functionality. (v0.0.252)
-// Upgrading `govuk-frontend-jinja==2.7.0` (the equivalent 4.7.0 version) does not fix it.
+// Upgrading `govuk-frontend-jinja==2.8.0` (the equivalent 4.7.0 version) does not fix it.
```
Comment (rayhan15243) [inline app/static/src/js/helpers.js:51]: @ramsharma-prog I don't think we need these comments anymore as we bumped to 2,8. Check with whoever made these comments in the code before removing it.

Comment (ramsharma-prog) [reply app/static/src/js/helpers.js]: Good point! removed @rayhan15243

Comment (rayhan15243) [reply app/static/src/js/helpers.js]: image
@ramsharma-prog

Comment (rayhan15243) [reply app/static/src/js/helpers.js]: 2.8 working as expected. Checked on Dev.

**Review by rayhan15243 — APPROVED** [review]

**Pull request merged**